rdrf/scripts/schema_test.py

```diff
@@ -0,0 +1,58 @@
+from report.schema.schema import create_dynamic_schema
+import json
+
+
+def get_results(schema, query):
```

Comment:
There are existing unit tests in `rdrf/rdrf/report/tests/schema_tests.py` with a few different scenarios and dummy data that might have been good to use to prove the dynamic schema and its outputs.

Reply:
Also the benefit of the unit test is that it creates its own data and there are no preconditions on the expected values of codes etc in the registry.

Reply:
Yep definitely, unit test is way better than my makeshift script :+1:. This was just about seeing if the idea even works with the existing registries.